kernel/distances/SSD.m

```diff
@@ -23,14 +26,17 @@
 % Input: 
 %  Tc, Rc     template and reference
 %  omega, m   represents domain and discretization
-%  varargin   optional parameters, e.g. doDerivative=0
+%  varargin   optional parameters, e.g. doDerivative=0, weights=1.0
```

Comment:
weights=1.0 is irritating:
If weights == 1.0 there is no point to add the weight